app/tetwild/auto_table.hpp

```diff
@@ -0,0 +1,36 @@
+// This file is part of fTetWild, a software for generating tetrahedral meshes.
+//
+// Copyright (C) 2019 Teseo Schneider <teseo.schneider@nyu.edu>
+// This Source Code Form is subject to the terms of the Mozilla Public License
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can
+// obtain one at http://mozilla.org/MPL/2.0/.
+//
+
+#pragma once
+
+#include <Eigen/Core>
+#include <array>
+#include <vector>
+
+namespace floatTetWild {
+typedef Eigen::Matrix<double, 3, 3> Matrix3;
+
+typedef Eigen::Matrix<double, 3, 1> Vector3;
```

Comment:
while different namespace, this is confusing with the other `tetwild::Vector3` which is rational